addons/analytic/views/analytic_distribution_model_views.xml

```diff
@@ -21,14 +21,18 @@
             <form string="Analytic Distribution Model">
                 <sheet>
                     <group>
-                        <group>
-                            <field name="partner_id"/>
-                            <field name="partner_category_id"/>
-                            <field name="company_id" groups="base.group_multi_company"/>
+                        <field name="analytic_distribution_stored_char" invisible="1"/>
+                        <group string="Simultaneous conditions to meet" colspan="2">
```

Comment:
Should not we have capital letters on every word here?  (Like Analytic Distribution Model) Maybe Simultaneous Conditions Needed.  I see we deviate from it in case of longer sentences, but maybe we can shorten it.

Reply:
group string is automatically uppercase IIANM